code/game/objects/items/binoculars.dm

```diff
@@ -146,19 +147,8 @@
 
 
 	var/turf/TU = get_turf(A)
-	var/area/targ_area = get_area(A)
 	if(!istype(TU))
 		return
-	var/is_outside = FALSE
```

Comment:
I'm in favor of expanding the places where lazes can happen, but not completely removing them.
There's deep caves in where we might still want to preserve the functionality.
Change the logic to a positive one, in where this becomes false if the ceiling is deep caves.

Reply:
It doesn't make sense for rangefinders not to work indoors. Rangefinders IRL don't calculate distance using some magical laser-finding satellite, but rather using their internal GPS as well as the time it takes for a signal to bounce back and forth between the designated target and the binoculars.

https://precisionrifleblog.com/2013/10/29/how-do-rangefinders-work/

Reply:
There's the rangefinder for coordinates for the mortar (which we don't use beyond that, because coordinates in this game is a pretty bad idea), and there's the laser pointer. They follow different logics, which are not entirely realistic.

The idea is to restrict some areas to be free from OBs and CAS, as safe zones. The problem is that 90% of the maps qualify for that, and the intention is to reduce that significantly.

That is why I ask for some kind of area not to be laze-able.

Reply:
CAS and OB are rarely used and when they are, the "attack" is super telegraphed. 

The mortar also has build in code to prevent any area that is deep underground from being targeted. As I said before anyone could just open the map in a map editor and get the coords themselves or just do basic estimation of the coordinate. For example, I always laze LZ1 and then subtract -30 from the value to get the center.

Reply:
Yes, this would be fine for the rangefinder mode to get coordinates (even thought that's something we want to remove in the long-term). The issue is with it generating lazes that can be interacted by CAS and OBs.